core/server/data/migrations/versions/4.17/03-add-offers-permissions.js

```diff
@@ -0,0 +1,42 @@
+const {
+    addPermissionWithRoles,
+    combineTransactionalMigrations
+} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
+    addPermissionWithRoles({
+        name: 'Browse offers',
+        action: 'browse',
+        object: 'offer'
+    }, [
+        'Administrator'
+    ]),
+    addPermissionWithRoles({
+        name: 'Read offers',
+        action: 'read',
+        object: 'offer'
+    }, [
+        'Administrator'
+    ]),
+    addPermissionWithRoles({
+        name: 'Edit offers',
+        action: 'edit',
+        object: 'offer'
+    }, [
+        'Administrator'
+    ]),
+    addPermissionWithRoles({
+        name: 'Add offers',
+        action: 'add',
+        object: 'offer'
+    }, [
+        'Administrator'
+    ]),
+    addPermissionWithRoles({
```

Comment:
Do we need/want this permission? AFAIK We're only going to implement archive not delete

Reply:
🤔 Hmm yeah thats true, can bring it back if/when needed, will remove it for now

Reply:
Removed